### PR TITLE
Add Workflow to remove the 'Tested' label

### PR DESCRIPTION
**Pull request by SaillantNicolas**
## Summary of Changes

Added a workflow that removes the `Tested` labels during a new commit in a pull request

## Release Management

* Affected package(s): -
* Issue(s) solved (if any): -
* Feature/Small Feature (if any): -

### CODE REVIEW

File: .github/workflows/Remove_labels.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          script: |
+            github.rest.issues.removeLabel({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              name: "Tested",
+            });
+      - name: Post address
+        uses: actions/github-script@v6
+        if: ${{ success() && github.event_name == 'pull_request_target' }}
```
Comment (lrineau) [inline .github/workflows/Remove_labels.yml:24]: Why the test `github.event_name == 'pull_request_target'`?

**Review by lrineau — APPROVED** [review]

**Comment by sloriot** [comment]
"Prevously tested" and "Ready to be tested" would indeed make sense.

**Comment by MaelRL** [comment]
"Ready to be tested" should denote a stable state that isn't guaranteed to be just because new stuff has been pushed.

If we were to automate labels a lot -- which I think is not worth the investment and maintenance time --, then it could be that it is at least passing all the automated checks (but it would still not be a good translation).

**Comment by albert-github** [comment]
I'm not sure what @sloriot  meant by setting the already existing label `Ready to be tested` but maybe he meant, at least I would interpret it as such: the "simple user" can set in one way or another this label.

@MaelRL you might probably be right about "which I think is not worth the investment and maintenance time", my comment was also just an idea so that people start to think about the possibilities (when not done yet) or start a brainstorm about it.

**Comment by albert-github** [comment]
Interesting point here is also that the "Tested" label is gone when the PR is synchronized with master (see e.g. https://github.com/CGAL/cgal/pull/6866#issuecomment-1335919831).

**Comment by MaelRL** [comment]
> I'm not sure what @sloriot meant by setting the already existing label Ready to be tested but maybe he meant, at least I would interpret it as such: the "simple user" can set in one way or another this label.

I don't know if it is very important if non-CGAL devs (I assume this is what you meant by "simple user") are able to set flags or not; in that case of "simple user", the "Ready to be tested" serves as a clear "some CGAL dev looked at it", which is useful.

> Interesting point here is also that the "Tested" label is gone when the PR is synchronized with master (see e.g. https://github.com/CGAL/cgal/pull/6866#issuecomment-1335919831).

Which is good, the merge commit is not necessarily trivial, especially if there's conflict resolution.

**Pull request merged**